app/jobs/replication_check_job.rb

```diff
@@ -0,0 +1,34 @@
+# Confirms existence of PreservedCopy on an endpoint.
+# Confirms the MD5 checksum matches in database and s3.
+class ReplicationCheckJob < ApplicationJob
+  # This queue is never expected to be used. For example:
+  # ReplicationCheck.set(queue: :endpoint_check_us_west_2).perform_later(pc)
+  # ReplicationCheck.set(queue: :endpoint_check_us_east_1).perform_later(pc)
+  queue_as :replication_check
+  delegate :bucket, :bucket_name, to: PreservationCatalog::S3
+
+  # @param [PreservedCopy] verify that the archived preserved_copy exists on an endpoint
+  def perform(preserved_copy)
+    return if preserved_copy.status == PreservedCopy::UNREPLICATED_STATUS # FIXME: storytime (Monday 07/02)
```

Comment:
Not important, but you can ask `preserved_copy.unreplicated?` via the enum and avoid the constant.